src/index.js

```diff
@@ -13,14 +13,15 @@
 // limitations under the License.
 
```

Comment:
Discard this file for the Pull Request because we have a backend setup for intents and we don't want such interference in the inde.js file.